spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/SpannerEntityWriter.java

```diff
@@ -38,6 +36,4 @@
 	 * written.
 	 */
 	void write(Object source, MultipleValueBinder sink, Set<String> includeColumns);
-
-	Key writeToKey(Object key);
```

Comment:
What's the benefit of pulling this out into the `SpannerKeyWriter` interface?

Reply:
The reason is this new one is what's added to the constructor of SpannerPersistentEntityImpl because it is needed to convert  the properties that comprise the composite key. We don't need the write-to-entity functionality, but only the write-to-key functionality.